app/src/androidTest/java/org/dhis2/usescases/teidashboard/TeiDashboardTest.kt

```diff
@@ -347,7 +347,9 @@ class TeiDashboardTest : BaseTest() {
 
         relationshipRobot {
             clickOnFabAdd()
+            waitToDebounce(500)
             clickOnRelationshipType()
+            waitToDebounce(500)
```

Comment:
not sure if you need this? Animations are disabled in Espresso. Did you try removing them?

Reply:
I tried, but could not make it work without this

Reply:
in your local machine or bitrise?